Backpack/Button/Classes/BPKButton.m

```diff
@@ -415,7 +416,11 @@ - (void)updateContentColor {
         highlightedContentColor = [self class].highlightedOutline;
         break;
     case BPKButtonStyleLink:
-        highlightedContentColor = [self.currentContentColor colorWithAlphaComponent:0.2];
+        if (self.primaryContentColor != nil) {
+            highlightedContentColor = [BPKColor blend:self.linkContentColor with:BPKColor.gray900 weight:0.85f];
```

Comment:
Can you expand on this logic? Isn't it enough to reduce opacity here?

Reply:
We already darken the colour for highlighted state. I think it should do the same when themed...

Reply:
Oh no we don't - I blind 😆

Reply:
All fixed @k0nserv